doc/source/conf.py

```diff
@@ -15,9 +15,19 @@
 
 import datetime
 import subprocess
+import warnings
+
+warnings.filterwarnings(
+    "ignore",
+    message="DCDReader currently makes independent timesteps",
+    category=DeprecationWarning,
+    module="MDAnalysis.coordinates.DCD",
+)
 
 import MDAnalysis as mda
 
+MDAnalysis_version = mda.__version__
+
```

Comment:
Can you please explain why this is necessary, wasn't the `:okwarning:` below enough?

Reply:
So this part addresses the warning - 

```
/home/runner/micromamba/envs/mda-user-guide/lib/python3.10/site-packages/MDAnalysis/coordinates/DCD.py:165: DeprecationWarning: DCDReader currently makes independent timesteps by copying self.ts while other readers update self.ts inplace. This behavior will be changed in 3.0 to be the same as other readers. Read more at https://github.com/MDAnalysis/mdanalysis/issues/3889 to learn if this change in behavior might affect you.
  warnings.warn("DCDReader currently makes independent timesteps"
```

For solving this warning changing `conf.py` file was needed. 

Adding `:okwarning:` helped to solve the other warning related to it.